README-fr.adoc

```diff
@@ -0,0 +1,310 @@
+L'inconvénient est que le paquet n'est pas forcément mis à jour immédiatement après la release de la gem.
+Si vous avez besoin de la dernière version, vous devez passer par la commande `gem`.
+
+=== (a) Installation de la gem
+
+Ouvrir un terminal et taper (en excluant le `$`) :
+
+ $ gem install asciidoctor
+
+Si vous souhaitez installer une version pre-release (càd, une release candidate), utilisez :
+
+ $ gem install asciidoctor --pre
+
+.Mettre à jour votre installation
+[TIP]
+====
+Si vous avez une précédente version d'Asciidoctor installée, vous pouvez la mettre à jour en utilisant :
+
+ $ gem update asciidoctor
+
+Si vous installez une nouvelle version de la gem en utilisant `gem install` au lieu de `gem update`, vous aurez plusieurs versions d'installées.
+Si c'est le cas, utilisez la commande gem suivante pour supprimer la vieille version :
+
+ $ gem cleanup asciidoctor
+====
+
+=== (b) Bundler
+
+. Créez un fichier Gemfile à la racine de votre projet (ou du répertoire courant),
+. Ajoutez la gem `asciidoctor` dans votre fichier Gemfile comme ci-dessous :
++
+[source]
+----
+source 'https://rubygems.org'
+gem 'asciidoctor'
+# ou spécifier la version explicitement
+# gem 'asciidoctor', '1.5.3'
+----
+
+. Sauvegardez le fichier Gemfile,
+. Ouvrez un terminal et installez la gem en utilisant :
+
+ $ bundle
+
+Pour mettre à jour la gem, spécifiez la nouvelle version dans le fichier GemFile et exécutez `bundle` à nouveau.
+Utiliser `bundle update` *n*'est *pas* recommandé car les autres gems seront également mises à jour, ce qui n'est pas forcément le résultat attendu.
+
+=== (c) Gestionnaire de paquets Linux
+
+==== Yum (Fedora 21 ou supérieure)
+
+Pour installer la gem sur Fedora 21 ou supérieure en utilisant yum, ouvrez un terminal et tapez :
+
+ $ sudo yum install -y asciidoctor
+
+Pour mettre à jour la gem, utilisez :
+
+ $ sudo yum update -y asciidoctor
+
+TIP: Votre système Fedora peut être configuré pour mettre à jour les paquets automatiquement, auquel cas aucune action de votre part ne sera nécessaire pour mettre à jour la gem.
+
+==== apt-get (Debian Sid, Ubuntu Saucy ou supérieure)
+
+Pour installer la gem sur Debian ou Ubuntu, ouvrez un terminal et tapez :
+
+ $ sudo apt-get install -y asciidoctor
+
+Pour mettre à jour la gem, utilisez :
+
+ $ sudo apt-get upgrade -y asciidoctor
+
+TIP: Votre système Debian ou Ubuntu peut être configuré pour mettre à jour les paquets automatiquement, auquel cas aucune action de votre part ne sera nécessaire pour mettre à jour la gem.
+
+=== Autres options d'installation
+
+* {uri-install-doc}[Installation de l'outillage Asciidoctor]
+* {uri-install-osx-doc}[Installation d'Asciidoctor sur Mac OS X]
+
+== Utilisation
+
+Si la gem Asciidoctor s'est installée correctement, l'interface de ligne de commande (CLI) `asciidoctor` sera disponible dans votre PATH.
```

Comment:
"interface de ligne de commande" is uncommon, we could use "ligne de commande" ?